lib/src/helpers/createComponentClassNames.js

```diff
@@ -0,0 +1,17 @@
+import cx from 'classnames';
+
+function createComponentClassNames(
```

Comment:
💚  I tried to unresolve and leave a comment on https://github.com/gathercontent/gather-ui/pull/884#discussion_r509112386 but it wouldnt let me so re-resolved it 😓 

But I agree with @altaudio I'm not sure if I do see the benefit of this, I find the old way a bit clearer to digest (though admittedly that could just be due to familarity)

Reply:
I'll revert to use `cx` and see if I can ignore `className` for propTypes.

Reply:
I've removed this and added to our eslint config 👍  (thanks for challenging this)